README.md

```diff
@@ -1,5 +1,5 @@
 # Eclipse Che documentation project
-[![Contribute](https://www.eclipse.org/che/contribute.svg)](https://che.openshift.io/f?url=https://github.com/eclipse/che-docs)
+[![Contribute](https://www.eclipse.org/che/contribute.svg)](https://workspaces.openshift.com/f?url=https://github.com/eclipse/che-docs)
```

Comment:
currently fails with `registry/v3/plugins/errata-ai/vale-server/latest/meta.yaml failed. Status code 404. Response body:
Not Found` 
@themr0c @rkratky unless someone fixes the docs devfile for `workspaces.openshift.com` the Contribute section will be removed

Reply:
Please open an issue if you found the devfile is not functionnal.

Reply:
Fix in progress: https://github.com/eclipse/che-docs/pull/1889.